### DISCUSSION TITLE
New Dirt bombs and Scarab bombs don't work

### DISCUSSION THREAD

**Discussion by albertttz** [Pryaxis/TShock#1865] [Q&A]
With the new journeys end update there came lots of new items. One of the items is a dirt bomb. When used it should explode with a pile of dirt. For some reason, it doesn't add the dirt after the explosion. All my permissions are enabled with addperm tshock.* and the config file doesn't seem to have anything with the new items. Any help with this would be great!

**Comment by hakusaro** [comment]
Should be somewhat fixed as of https://github.com/Pryaxis/TShock/commit/d610ab929b748713da393dc50cbfb153b5a51c5f and you can expect to see this in the next pre-release of TShock 4.4.0.
